pom.xml

```diff
@@ -57,7 +57,7 @@
         <failsafe.version>2.18.1</failsafe.version>
         <slf4j.api.version>1.7.5</slf4j.api.version>
 
-        <koryphe.version>1.0.0</koryphe.version>
+        <koryphe.version>1.0.1-SNAPSHOT</koryphe.version>
```

Comment:
This PR cannot be merged in until Koryphe 1.1.0 has been released. We cannot depend on SNAPSHOT versions.